python/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "datafusion-bigtable-python"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "_internal"
+crate-type = ["cdylib"]
+
+[package.metadata.maturin]
+name = "datafusion_bigtable._internal"
+
+[dependencies]
+pyo3 = { version = "~0.15", features = ["extension-module"] }
+tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync"] }
+datafusion = { version = "^7.0.0", features = ["pyarrow"] }
+datafusion-python = { git = "https://github.com/jychen7/datafusion-python", branch = "register-table" }
```

Comment:
switch to https://github.com/datafusion-contrib/datafusion-python after PR https://github.com/datafusion-contrib/datafusion-python/pull/46